controllers/cinder_controller.go

```diff
@@ -911,6 +933,24 @@ func (r *CinderReconciler) schedulerDeploymentCreateOrUpdate(ctx context.Context
 }
 
 func (r *CinderReconciler) backupDeploymentCreateOrUpdate(ctx context.Context, instance *cinderv1beta1.Cinder) (*cinderv1beta1.CinderBackup, controllerutil.OperationResult, error) {
+
+	svcTemplate := instance.Spec.CinderBackup
+	cinderBackupSpec := cinderv1beta1.CinderBackupSpec{
+		ContainerImage:             svcTemplate.ContainerImage,
+		Replicas:                   svcTemplate.Replicas,
+		Debug:                      svcTemplate.Debug,
+		CustomServiceConfig:        svcTemplate.CustomServiceConfig,
+		CustomServiceConfigSecrets: svcTemplate.CustomServiceConfigSecrets,
+		DefaultConfigOverwrite:     svcTemplate.DefaultConfigOverwrite,
+		NetworkAttachments:         svcTemplate.NetworkAttachments,
+		ServiceUser:                instance.Spec.ServiceUser,
+		DatabaseHostname:           instance.Status.DatabaseHostname,
+		DatabaseUser:               instance.Spec.DatabaseUser,
+		PasswordSelectors:          instance.Spec.PasswordSelectors,
+		Secret:                     instance.Spec.Secret,
+		ExtraMounts:                instance.Spec.ExtraMounts,
+		TransportURLSecret:         instance.Status.TransportURLSecret,
```

Comment:
same as above about the externalEndpoints

Reply:
(see comment above)